authbridge/demos/github-issue/rbac/Makefile

```
@@ -0,0 +1,259 @@
+#   make clean                 # delete all deployed resources
+#
+# Prerequisites:
+#   - uv (https://docs.astral.sh/uv/)
+#   - kubectl pointing at the Kind cluster
+#   - Kind cluster accessible (default name: kagenti)
+#   - Keycloak running at http://keycloak.localtest.me:8080
+#   - AGENT_EXAMPLES_DIR set to the agent-examples repo clone (for build-images)
+#   - PRIVILEGED_ACCESS_PAT and PUBLIC_ACCESS_PAT set for the deploy target
+
+.PHONY: help preflight build-images setup-namespace setup-keycloak apply-configmaps \
+        check-pats create-secret deploy-tool deploy-agent deploy validate test test-rbac clean
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)